lib/constants.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Application constants.
+ * @type {{DIRECTORY_APP: string, DIRECTORY_APP_LAYOUTS: string, DIRECTORY_APP_PAGES: string,
```

Comment:
I checked with IntelliJ IDEA, IDE can resolve these constants without the `@type {{ ... }}` tag. So we can remove `@type` tag from this comment.